src/components/Uploader.js

```diff
@@ -246,16 +244,12 @@ class MuxVideoInputUploader extends Component {
         </FormField>
         <ButtonCollection>
           <FileInputButton
-            inverted
-            icon={UploadIcon}
+            icon={<FiUpload data-sanity-icon="upload" />}
             onSelect={(files) => this.handleUploadFile(files[0])}
             accept={'video/*'}
```

Comment:
same comment here about `accept=`video/*'`

Reply:
Removed in 2ee7c22